RocketApp/Features/Sources/RocketList/RocketListCore.swift

```diff
@@ -1,43 +1,97 @@
-import ComposableArchitecture
+import RocketListCell
 import RocketDetail
+import NetworkClientExtensions
 import RocketsClient
+import ComposableArchitecture
+import Foundation
+import NetworkingCoreToolkit
 
-public struct RocketListState: Equatable {
-  var rockets: [RocketDetail]
+public struct RocketListCore: ReducerProtocol {
+  public struct State: Equatable {
+    var rocketsData: IdentifiedArrayOf<RocketListCellCore.State> {
```

Comment:
Tohle bych teda nedelal, uplne zbytecne narusujes single source of truth. Mas `loadingStatus` a to by ti melo stacit. `switch` s tim umi dobre pracovat.

Reply:
Tak si to trochu musis postudovat ;)

Reply:
Ale kdybys z toho byl zase zoufalej, tak to zavri, ale nemyslim si, ze by to melo bejt potreba.

Reply:
Udelal jsem extension na Loadable  s writable computed property ktera pracuje primo s IdentifiedArray. Zbavil jsem se tak property rocketsData. V budoucnu ji predelam na generickou funkci a presunu do Loadable, pokud to pujde.